SmartDeviceLink/SDLStreamingMediaLifecycleManager.m

```diff
@@ -393,6 +408,23 @@ - (void)didEnterStateVideoStreamReady {
     }
 
     [[NSNotificationCenter defaultCenter] postNotificationName:SDLVideoStreamDidStartNotification object:nil];
+
+    if (self.useDisplayLink) {
+        dispatch_async(dispatch_get_main_queue(), ^{
+            // And start up the displayLink
+            NSInteger targetFramerate = ((NSNumber *)self.videoEncoderSettings[(__bridge NSString *)kVTCompressionPropertyKey_ExpectedFrameRate]).integerValue;
+            SDLLogD(@"Initializing CADisplayLink with framerate: %ld", (long)targetFramerate);
+            self.displayLink = [CADisplayLink displayLinkWithTarget:self selector:@selector(sdl_displayLinkFired:)];
+            if (SDL_SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"10")) {
+                self.displayLink.preferredFramesPerSecond = targetFramerate;
+            } else {
+                self.displayLink.frameInterval = (60 / targetFramerate);
+            }
+            [self.displayLink addToRunLoop:[NSRunLoop currentRunLoop] forMode:NSRunLoopCommonModes];
+        });
```

Comment:
I am having issues with the video frame going blurry every few seconds when nothing on the screen is changing. For example if I have an image on the SDL screen, it is relatively crisp but a second later, it goes blurry, and then back to being sharp again.

Reply:
Interesting. I assume that this is related to new restrictions that were added on the amount of data allowed to be sent per 5 seconds. This limit may have been too low.

Reply:
It is very noticeable as images and text seem to flash and pulse every few seconds.

Reply:
I tested the **up the max data rate limit / 5 second** change and it did not fix the flickering.

Reply:
Removed the data limit

Reply:
Removing the hard data limit fixes the flickering issue.